frontend/src/app/libs/admin-role-management/store/users/users.actions.ts

```diff
@@ -0,0 +1,37 @@
+import { Action } from '@ngrx/store';
+import { User } from 'src/app/libs/auth/models/user';
+
+export enum UsersActionType {
+	LoadUsers = '[Users] Load Users',
+	LoadUsersSuccess = '[Users] Load Users Success',
+	LoadUsersFail = '[Users] Load Users Fail',
+
+	UpdateUser = '[Users] Update user',
```

Comment:
no need to create update user action since this api already provided by `AuthFacade`